### ISSUE TITLE
Decreased selenium level

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5951]
**Preferred term label:** Decreased selenium level

**Synonyms**

**Definition (free text, please give PubMed ID)** A reduced concentration of selenium in the blood., PMID:25519493

**Parent term (use hpo.jax.org/app)** Abnormal circulating selenium concentration HP:0031903

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added

**Comment by JillMiller** [comment]
Hi Peter. I am unable to find this term on GitHub, do you know what number it was added as?

**Comment by nicolevasilevsky** [comment]
@JillMiller 
I think it is:
HP_0033192 'Decreased circulating selenium concentration'
